newton/tests/test_collision_pipeline.py

```
@@ -454,6 +454,67 @@ def test_mesh_mesh_bvh_vs_bvh(_test, device, broad_phase_mode: newton.BroadPhase
+            pair = (min(s0, s1), max(s0, s1))
+            test.assertNotEqual(
+                pair,
+                excluded,
+                f"Excluded pair {excluded} must not appear in contacts (broad_phase={broad_phase_mode.name})",
+            )
+        # With the only pair excluded, we must have zero rigid contacts
+        test.assertEqual(n, 0, f"Expected 0 rigid contacts when only pair is excluded (got {n})")
+
+
+add_function_test(
+    TestUnifiedCollisionPipeline,
```

Ruff (0.14.14)

[error] Undefined name `TestUnifiedCollisionPipeline`

(F821)